app/src/components/Main/Main.js

```diff
@@ -149,6 +162,7 @@ function Main(props) {
           <input
             className="popup__submit"
             disabled={signInState}
+            onClick={handleSignIn}
```

Comment:
please, delete it

* Submit handlers are added only to tags `form` with event `submit` rather than to submit buttons with event `click` because it also automatically handles `Enter` presses